### PR TITLE
Added thexyz

### PR DESCRIPTION
**Pull request by ptoone**
Changed category to partial as requested.

### CODE REVIEW

File: _data/hosting_providers.json [hunk 1]
```diff
@@ -2188,5 +2188,15 @@
     "plan": "https://www.easyname.com/en/hosting/free-lets-encrypt-ssl-certificate",
     "reviewed": "2019.7.10",
     "note": ""
+  }
+     {
+    "name": "Thexyz",
+    "link": "https://www.thexyz.com",
+    "category": "partial",
+    "tutorial": "https://www.thexyz.com/account/knowledgebase/117/How-do-I-install-a-Lets-Encrypt-SSL-certificate-on-my-site.html",
+    "announcement": "[No need to find this]",
```
Comment (ohemorange) [inline _data/hosting_providers.json]:
```suggestion
    "announcement": "",
```

This content is a holdover from a previous version.

File: _data/hosting_providers.json [hunk 2]
```diff
@@ -2188,5 +2188,15 @@
...
+    "plan": "",
+    "reviewed": "",
```
Comment (ohemorange) [inline _data/hosting_providers.json]: Let's add today's date so we know when this was last updated.
```suggestion
    "reviewed": "2019.7.22",
```

File: _data/hosting_providers.json [hunk 3]
```diff
@@ -2188,5 +2188,15 @@
     "plan": "https://www.easyname.com/en/hosting/free-lets-encrypt-ssl-certificate",
     "reviewed": "2019.7.10",
     "note": ""
+  }
+     {
```
Comment (ohemorange) [inline _data/hosting_providers.json]: One teeeny tiny nit, then we're good to go!
```suggestion
  {
```

File: _data/hosting_providers.json [hunk 4]
```diff
@@ -2188,5 +2188,15 @@
     "plan": "https://www.easyname.com/en/hosting/free-lets-encrypt-ssl-certificate",
     "reviewed": "2019.7.10",
     "note": ""
+  }
```
Comment (ohemorange) [inline _data/hosting_providers.json]:
```suggestion
  },
```

**Comment by ptoone** [comment]
It does not appear to be added though? Is there something else I need to do?

**Comment by ohemorange** [comment]
It's there if you click on "partial HTTPS support", looks like the search box is broken though. Thanks for helping us catch that!

**Comment by ptoone** [comment]
I don't see it. Can you share the link of that page?

**Comment by ohemorange** [comment]
For now, go to https://certbot.eff.org/hosting_providers/ then click "partial HTTPS support"

**Pull request merged**